roles/foreman/tasks/main.yaml

```diff
@@ -75,6 +75,8 @@
     state: quadlet
     network: host
     hostname: "{{ ansible_fqdn }}"
+    restart_policy: "on-failure"
+    restart_sec: 1
```

Comment:
If I read https://docs.ansible.com/ansible/latest/collections/containers/podman/podman_container_module.html#ansible-collections-containers-podman-podman-container-module right then it needs to be under `generate_systemd`. I think that's what lint is complaining about.

Reply:
You are right.. my window was too tiny to see the options were nested.